packages/foam-vscode/src/features/janitor.ts

```diff
@@ -71,7 +70,7 @@ async function janitor(foam: Foam) {
 }
 
 async function runJanitor(foam: Foam) {
-  const notes: Note[] = foam.workspace.list().filter(isNote);
+  const notes: Resource[] = foam.workspace.list();
```

Comment:
If we already have attachment resources in the workspace, I bet the janitor doesn't know what to do with them

Reply:
what's the janitor?

... sorry I couldn't resist hehe

but yes, here we would probably need to filter by type (it's a bit of a coupling, but md is still the "native" resource in Foam, so I think it's fine - I don't think it's worth changing the provider api to make the janitor happy)